quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,141 @@
+
+## Why do I need a quickstart?
+
+A quickstart is often the first opportunity for your user to build an opinion about the technical quality of your project. It can:
+- Make users comfortable using the application.
+- Work with any application version the user might use.
+- Lead to the willingness to handle complex workflows.
+- Help the users in exploring additional features.
+- Improve the customer experience, and help reduce costs by lowering the number of support requests.
+
+## Tips for writing a quickstart
+
+- Highlight the primary feature of your application.
+- Document the quickest/easiest way to implement the primary feature  of your application.
+- Pick a use case that the user can complete within 1-2 hours with a preference for a shorter time.
+-  Avoid complicating the quickstart by including error scenarios/complex use cases.
+- Lengthy quickstarts can overwhelm users. Consider condensing or removing steps or reevaluating the scope of the quickstart.
+- For code samples, ensure that you include:  
+ - Any required `import` or `using` statements.
+ - Code comments that explain what the code does.
+
+## About the "Metadata" fields
+
+For more information on how to update the metadata, see [Document version](../base/base-guide.md#document-version), [Last Updated](../base/base-guide.md#last-updated), and [Application Version](../base/base-guide.md#application-versions) on the Good Docs Projects Base template.
+
+## About the "Overview" section
+
+Use this section to provide the following:
+
+- A short description of your application and its purpose.
+- A description of what the user can accomplish in this quickstart.
+- The intended audience for this document. 
+- The basic knowledge that you expect the user to have before using this quickstart.
+
+Defining the audience right in the beginning helps the users identify if the quickstart is relevant for their usage.
+
+## About the "Before you begin" section
+
+This section is optional.
+
+Use this section to mention any requirements/configuration prerequisites that the user must complete before they start the quickstart. 
+For easy understanding, we recommend that you classify the prerequisites into different categories such as software prerequisites, hardware prerequisites, and so on. 
+
+
+
+## About the "Installation" section
+
+This section is optional.
+
+:memo: Not all quickstart guides requires installation section. Include this section if:
+      - Installation and/or configuration is  done at the same time, and by the same person running the quickstart.
+      - Installation of a specific software(s) are a prerequisite to running the quickstart. 
+ 
+The purpose of this section is to provide instructions to your users on how to install/configure a particular software/tool before running the quickstart. 
+
+:memo: This section may not be relevant for Cloud/API-based applications where authorization/authentication information is more applicable.
```

Comment:
**Nitpick (formatting, non-blocking):** Rather than use a slash (/) mark, use the conjunction "or" instead or rephrase it. For example: "...how to install or configure a particular tool before running..." and "where authorization or authentication information is..."